crates/holochain_state/src/mutations.rs

```diff
@@ -583,10 +583,14 @@ pub fn insert_entry(txn: &mut Transaction, entry: EntryHashed) -> StateMutationR
 /// because the chain is locked if there are ANY locks that don't match the
 /// current id being queried.
 /// In practise this is useless so don't do that. One lock at a time please.
-pub fn lock_chain(txn: &mut Transaction, lock: &[u8], end: &Timestamp) -> StateMutationResult<()> {
+pub fn lock_chain(
+    txn: &mut Transaction,
+    lock: &[u8],
+    expires_at_ms: &Timestamp,
```

Comment:
little pedantic, but this param is not actually ms, since it's a Timestamp type...